README.md

```diff
@@ -15,9 +15,27 @@ from scratch as well as keep it up-to-date over time.
 
 ### Prerequisites
 
+* `brew`
+
+```bash
+xcode-select -–install
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+```
+
+>see: [Install Homebrew without XCODE](https://www.codeandpeace.com/install-homebrew-without-xcode/)
+
 * `git` must be installed
+
+```bash
```

Comment:
when i initially added this note, my thought was to leave it up to the reader with the assumption that the system installed `git` would usually be enough (such as in osx). the system is upgraded to the brew installed one as part of the installation script.

since brew is osx specific, does the assumption of there being a system installed version on a new machine still hold? if it makes sense to add this, but i think it would be helpful to add some instruction that suggests the command is only for osx, like "if the new machine is running macOS and does not have an existing git installation, you can use brew to install one" (but this assumes brew is already installed in the chicken and egg new machine problem :))